app/src/test/java/org/oppia/android/app/testing/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -81,9 +77,15 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
+private const val SMALL_TEXT_SIZE_SCALE = 0.8f
+private const val MEDIUM_TEXT_SIZE_SCALE = 1.0f
+private const val LARGE_TEXT_SIZE_SCALE = 1.2f
+private const val EXTRA_LARGE_TEXT_SIZE_SCALE = 1.4f
+
 private const val SMALL_TEXT_SIZE = 0
-private const val MEDIUM_TEXT_SIZE = 5
-private const val LARGE_TEXT_SIZE = 10
+private const val MEDIUM_TEXT_SIZE = 1
```

Comment:
we are using these as an index, so can we name these as `MEDIUM_TEXT_SIZE_INDEX`?

Reply:
done